test/project_types/script/ui/error_handler_test.rb

```diff
@@ -232,12 +213,23 @@ def should_call_display_and_raise
       describe "when ScriptConfigurationDefinitionError" do
         let(:err) do
           Script::Layers::Infrastructure::Errors::ScriptConfigurationDefinitionError.new(
-            message: "message",
+            messages: messages,
             filename: "filename",
           )
         end
-        it "should call display_and_raise" do
-          should_call_display_and_raise
+
+        describe "when there is a single error message" do
+          let(:messages) { ["message"] }
+          it "should call display_and_raise" do
+            should_call_display_and_raise
+          end
+        end
+
+        describe "when there are multiple error messages" do
+          let(:messages) { ["message1", "message2"] }
+          it "should call display_and_raise" do
+            should_call_display_and_raise
+          end
         end
```

Comment:
Should we assert that it's calling with the correct translation keys here?

Reply:
I agree that it could be worthwhile to check that the right keys are sent but that feels beyond the scope of this PR, as I'm just following the pattern in this file.